OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/CourseModalDialogFragment.kt

```diff
@@ -121,6 +133,31 @@ class CourseModalDialogFragment : DialogFragment() {
             })
     }
 
+    private fun fetchProductPrice() {
```

Comment:
The function name is atomic but it's doing two tasks:
- Fetching course price
- Setting prices

Maybe we can rename it or have two different functions

Reply:
Renamed with `initializeProductPrice`.